[2021-03-30 07:59:46,677] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.bash3 2021-03-28T00:00:00+00:00 [queued]>
[2021-03-30 07:59:46,752] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.bash3 2021-03-28T00:00:00+00:00 [queued]>
[2021-03-30 07:59:46,752] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-30 07:59:46,752] {taskinstance.py:1018} INFO - Starting attempt 4 of 4
[2021-03-30 07:59:46,752] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-30 07:59:46,795] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): processing_tasks.bash3> on 2021-03-28T00:00:00+00:00
[2021-03-30 07:59:46,803] {standard_task_runner.py:51} INFO - Started process 885971 to run task
[2021-03-30 07:59:46,832] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'processing_tasks.bash3', '2021-03-28T00:00:00+00:00', '--job-id', '209', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp2jofio35']
[2021-03-30 07:59:46,850] {standard_task_runner.py:76} INFO - Job 209: Subtask processing_tasks.bash3
[2021-03-30 07:59:47,105] {logging_mixin.py:103} INFO - Running <TaskInstance: parallel_dag.processing_tasks.bash3 2021-03-28T00:00:00+00:00 [running]> on host airflowvm
[2021-03-30 07:59:47,365] {taskinstance.py:1230} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.bash3
AIRFLOW_CTX_EXECUTION_DATE=2021-03-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-28T00:00:00+00:00
[2021-03-30 07:59:47,365] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-30 07:59:47,376] {bash.py:158} INFO - Running command: sleep 3
[2021-03-30 07:59:47,391] {bash.py:169} INFO - Output:
[2021-03-30 07:59:50,401] {bash.py:177} INFO - Command exited with return code 0
[2021-03-30 07:59:50,477] {taskinstance.py:1135} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=processing_tasks.bash3, execution_date=20210328T000000, start_date=20210330T075946, end_date=20210330T075950
[2021-03-30 07:59:50,556] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-30 07:59:50,563] {local_task_job.py:118} INFO - Task exited with return code 0
